src/js/components/projects/detail.tsx

```diff
@@ -224,6 +227,25 @@ const ProjectDetail = (props: RouteComponentProps) => {
   const closeEditModal = () => {
     setEditModalOpen(false);
   };
+  // "delete" modal related:
+  const openDeleteModal = () => {
+    setDeleteModalOpen(true);
+  };
+  const closeDeleteModal = () => {
+    setDeleteModalOpen(false);
+  };
+  const handleProjectDelete = useCallback(() => {
+    if (project) {
+      dispatch(
+        deleteObject({
+          objectType: OBJECT_TYPES.PROJECT,
+          object: project,
+        }),
+      ).finally(() => {
+        closeDeleteModal();
+      });
+    }
+  }, [dispatch, project]);
```

Comment:
@jgerigmeyer I'm on the fence about whether this should just be handled in the modal instead.  Curious what you think!